### PR TITLE
PODS-2200 : Create feature switch for UserAnswers Transformation in pension-administrator

### PR DESCRIPTION
**Pull request by abhijeetardale**
PODS-2200 : Create feature switch for UserAnswers Transformation in pension-administrator

### CODE REVIEW

File: app/config/AppConfig.scala [hunk 1]
```diff
@@ -51,5 +51,6 @@ class AppConfig @Inject()(runModeConfiguration: Configuration, environment: Envi
   lazy val invitationCallbackUrl: String = s"$baseUrlPensionAdministrator${runModeConfiguration.underlying.getString("serviceUrls.invitation.callback")}"
 
   lazy val isManualIVEnabled: Boolean = runModeConfiguration.getBoolean("features.is-iv-enabled").getOrElse(false)
+  lazy val IsVariationsEnabled: Boolean = runModeConfiguration.getBoolean("features.is-variations-enabled").getOrElse(false)
```
Comment (grantrobson) [inline app/config/AppConfig.scala]: Should add to Toggles.scala too

File: app/config/AppConfig.scala [hunk 2]
```diff
@@ -51,5 +51,6 @@ class AppConfig @Inject()(runModeConfiguration: Configuration, environment: Envi
   lazy val invitationCallbackUrl: String = s"$baseUrlPensionAdministrator${runModeConfiguration.underlying.getString("serviceUrls.invitation.callback")}"
 
   lazy val isManualIVEnabled: Boolean = runModeConfiguration.getBoolean("features.is-iv-enabled").getOrElse(false)
```
Comment (grantrobson) [inline app/config/AppConfig.scala]: Also, not sure you need to add to appconfig now. Only needs to go into featureswitchmanagementservice now

File: app/config/AppConfig.scala (hunk 1 above)
Comment (abhijeetardale) [reply app/config/AppConfig.scala]: fixed

File: app/config/AppConfig.scala (hunk 2 above)
Comment (abhijeetardale) [reply app/config/AppConfig.scala]: removed as not required

**Pull request merged**